addons/pos_razorpay/static/src/app/payment_razorpay.js

```diff
@@ -82,7 +83,7 @@ export class PaymentRazorpay extends PaymentInterface {
 
     _process_razorpay(cid) {
         const order = this.pos.get_order();
-        const line = order.paymentlines.find((paymentLine) => paymentLine.cid === cid);
+        const line = order.get_selected_paymentline();
```

Comment:
Quick question. Why do we send the cid to the method if we find the payment line by using get_selected_paymentline? I saw that this also happens in other modules and I guess it is fine like this but is there a way to get through this method and the selected orderline is not the razorpay one ?